src/cipher.rs

```diff
@@ -40,39 +40,20 @@ impl CipherText {
         vec_bytes.to_base64(STANDARD)
     }
 
-    pub fn from_file(path: &str) -> Result<CipherText, Error> {
-        let mut text = String::new();
-        let file = try!(File::open(Path::new(path)).map_err(|e| Error::File(e.to_string())));
-        let in_file = BufReader::new(file);
-        for line in in_file.lines() {
-            text.push_str(&line.unwrap());
-        }
-        CipherText::from_b64(&text)
+    fn from_file<P: AsRef<Path>>(path: P) -> Result<CipherText, Error> {
+        buffer_file(path.as_ref()).and_then(CipherText::from_b64)
```

Comment:
I have a question about this. In the `to_file()` method we add new lines every 80 characters but we don't take that into account while reading the file. I think this works because the base64 decoder is ignoring the newline char, is this correct?